roles/sap_ha_install_hana_hsr/tasks/configure_firewall.yml

```diff
@@ -1,60 +1,19 @@
 ---
```

Comment:
This should be separete role as mentioned in another comment.

Reply:
Agreed, removed.

Reply:
Tracked under GH Issue #168